projects/api-proxy/proxy-server.js

```diff
@@ -174,6 +174,17 @@ app.use('/sormas-rest/dashboard/newCases', (req, res) => {
   });
 });
 
+app.use('/sormas-rest/dashboard/newEvents', (req, res) => {
```

Comment:
no need to hardcode the response, it works (for new cases too)

Reply:
Resolved.